src/process_impl/session.rs

```diff
@@ -10,23 +10,24 @@ use tokio::process;
 
 use tempfile::TempDir;
 
+/// Session does not have field "addr" because ssh don't care about
+/// the addr as long as we have the ctl.
+/// It is tested on OpenSSH 8.2p1, 8.9p1, 9.0p1
```

Comment:
I don't think we need this comment really.

Reply:
This comment is removed and I added that to `new_std_cmd` where it is actually relevant.